net.lutris.Lutris.yml

```diff
@@ -56,6 +56,16 @@ add-extensions:
     no-autodownload: true
     autodelete: true
 
+  com.valvesoftware.Steam.Utility:
+    subdirectories: true
+    directory: utils
+    version: beta
+    versions: beta;test
+    add-ld-path: lib
+    merge-dirs: share/vulkan/explicit_layer.d;share/vulkan/implicit_layer.d;
+    no-autodownload: true
+    autodelete: true
```

Comment:
I guess it should be `autodelete: false`, otherwise removing Lutris will remove Steam's extensions.